app/sass/partials/_feature-links.scss

```diff
@@ -43,17 +38,23 @@
 }
 
 .feature-link__title {
+  top: 85px;
```

Comment:
It's better to have parameters in alphabetical order for easy searching/glancing, unless there are multiple parameters that go together like margin/padding, width/height, top/left, bottom/right.

Reply:
top goes with `postion: relative`